File: tests/fixtures/code_samples/java/sample_017.java
// Sample 17: small utility.
package samples;

import java.util.List;

public final class Sample017 {
    private Sample017() {}

    public static int operation(List<Integer> xs) {
        int total = 17;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 17) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_017.py
"""Sample 17: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_17(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 17."""
    total = 17
    for x in xs:
        total += int(x)
    return total


def operation_17_pure(value: int) -> int:
    return (value * 17) % 7919

